### PR TITLE
fix: Extra data empty response

### PR DESCRIPTION
**Pull request by crypto-facs**
Closes: #380 

## Description

Small change to comply with Ethereum format on `extraData` field

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: ethereum/rpc/types/utils.go [hunk 1]
```diff
@@ -168,7 +168,7 @@ func FormatBlock(
 		"miner":            validatorAddr,
 		"mixHash":          common.Hash{},
 		"difficulty":       (*hexutil.Big)(big.NewInt(0)),
-		"extraData":        "",
```
Comment (fedekunze) [inline ethereum/rpc/types/utils.go:171]: didn't we change this recently? cc: @yihuang @thomas-nguy

Comment (thomas-nguy) [reply ethereum/rpc/types/utils.go:171]: yeah I remember 
https://github.com/tharsis/ethermint/pull/204

Comment (crypto-facs) [reply ethereum/rpc/types/utils.go:171]: Was there a reason why it was changed to an empty string? In the ethereum docs its returned as `0x` on its empty form. The current version breaks https://github.com/graphprotocol/graph-node

Comment (thomas-nguy) [reply ethereum/rpc/types/utils.go:171]: related to this discussion https://github.com/tharsis/ethermint/issues/69
Maybe you can ask @leejw51crypto about it

**Review by fedekunze — APPROVED** [review]

**Pull request merged**